library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/SessionReplayPrivacyTest.kt

```diff
@@ -87,13 +89,16 @@ internal class SessionReplayPrivacyTest {
         private val mockButtonMapper: ButtonMapper = mock()
         private val mockEditTextViewMapper: EditTextViewMapper = mock()
         private val mockImageMapper: ViewScreenshotWireframeMapper = mock()
+        private val mockUnsupportedViewMapper: UnsupportedViewMapper = mock()
 
         private val baseMappers = listOf(
-                MapperTypeWrapper(Button::class.java, mockButtonMapper.toGenericMapper()),
-                MapperTypeWrapper(EditText::class.java, mockEditTextViewMapper.toGenericMapper()),
-                MapperTypeWrapper(ImageView::class.java, mockImageMapper.toGenericMapper()),
-        )
+            MapperTypeWrapper(Button::class.java, mockButtonMapper.toGenericMapper()),
+            MapperTypeWrapper(EditText::class.java, mockEditTextViewMapper.toGenericMapper()),
+            MapperTypeWrapper(ImageView::class.java, mockImageMapper.toGenericMapper()),
 
+            MapperTypeWrapper(AppCompatToolbar::class.java, mockUnsupportedViewMapper.toGenericMapper()),
+            MapperTypeWrapper(Toolbar::class.java, mockUnsupportedViewMapper.toGenericMapper())
```

Comment:
shouldn't you add a test case for when the SDK is above `LOLLIPOP` ? The way I see it here you are not testing that `if` branch because you are adding this mapper in the `baseMappers` by default.

Reply:
Added additional testing for Kitkat